File: My-Resturant-Frontend/src/app/interFaces/Item.ts
export interface item{
  id:number,
  name:string,
  description:string,
  price:number,
  category:string|null,
  image:string|null,
  ingrediants:Array<string>,
  stock:number,
  modificationDate:Date,
  isActive:boolean
}
export interface editItem{
  name:string|null,
  description:string|null,
  price:number|null,
  category:string|null,
  image:string|null,
  ingrediants:Array<string>|null,
  stock:number|null,
  isActive:boolean|null
}
export interface cartItem{
  id:number,
  description:string,
  name:string,
  image:string|null,
  price:number,
  quantity:number
}

File: My-Resturant-API/Entities/Item.cs
namespace My_Resturant.Entities
{
    public class Item:MainEntity
    {
        public string name {  get; set; }
        public string description { get; set; }
        public double price { get; set; }
        public int? category { get; set; }
        public string? image { get; set; }
        public List<string> ingrediants { get; set; }
        public int stock { get; set; }
    }
}
